src/main/java/frc/robot/subsystems/Climber2020.java

```diff
@@ -1,14 +1,14 @@
 package frc.robot.subsystems;
 
-import com.revrobotics.CANSparkMax;
-import com.revrobotics.CANSparkMaxLowLevel.MotorType;
-
 import edu.wpi.first.wpilibj.smartdashboard.SendableBuilder;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.RobotConstants;
+import frc.robot.drive.MotorType;
+import frc.robot.drive.SpeedControllerEncoder;
+import frc.robot.drive.SpeedControllerFactory;
```

Comment:
It's a code smell that we're reaching into the `drive` package here. These should probably be moved somewhere more generic, maybe a `motors` package.

Reply:
Once when we get the base subsystems done, we can do a full refactor